### DISCUSSION TITLE
Wyszukiwanie ofert z produktami

### DISCUSSION THREAD

**Discussion by jaceklodz** [allegro/allegro-api#4336] [Suggestion - verification]
Cześć
czy zasób /sale/offers nie mógły zwracać product.id skoro można filtrować po product.id.empty ?

**Comment by MartaNowaczyk** [comment]
Cześć, dla GET /sale/offers/ zwracane wartości w REST API odpowiadają zakładce "Moje oferty" na froncie, dlatego product.id nie ma na liście ofert w REST API.  
Twoją sugestię dodania product.id dopiszę do listy sugestii dotyczącej rozwoju REST API.
